base/steer/FairRootManager.h

```diff
@@ -183,7 +183,8 @@ class FairRootManager : public TObject
     /** Replace the time based branch name list*/
     void SetTimeBasedBranchNameList(TList* list);
 
```

Comment:
I wonder, whether CBM could either call FillEventHeader on the Source directly (after all you likely setup the source, so you have that reference anyway?), or could use FairRun::FillEventHeader (then we should make it public)?

Reply:
The second option isn't possible. If we can use the function from the source directly I have to check but probably it will work. Will check next week.

Reply:
1. I can also leave out the deprecation for this PR, if that makes things easier?

2. What about this for your code as an intermediate step?

   ```cpp
   auto source = rootMgr->GetSource();
   if (source) {
       source->FillEventHeader(fEvtHeader);
   }
   ```

Reply:
I checked and CBM is able to use the sink directly so no more objections from my side.